contracts/HubPool.sol

```diff
@@ -194,6 +196,7 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
         address indexed proposer
     );
     event RootBundleExecuted(
+        bool relayToSpokePool,
```

Comment:
does the ordering in this event impact the gas cost?

Reply:
It does but I can't get the gas lower than putting it here. I tried placing it in other spots